scrawl_be/settings.py

```diff
@@ -88,7 +91,17 @@
     }
 }
 
-
+REST_FRAMEWORK = {
+    'DEFAULT_RENDERER_CLASSES': (
+        'rest_framework.renderers.JSONRenderer',
+    ),
+    'DEFAULT_PARSER_CLASSES': (
+        'rest_framework.parsers.JSONParser',
+    ),
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.AllowAny',
```

Comment:
What is this allowing?

Reply:
This is a CORS thing, I set it at the moment to just allow any request. We should probably change it later.